includes/spvgenerators.hexpat

```diff
@@ -0,0 +1,33 @@
+enum GeneratorID : u16 {
```

Comment:
Are these IDs and opcodes relevant for other data types too? If not, I'd put them into the main file. Includes are meant as sort of small libraries of types (and later on functions) that can be included and used by other people.

Reply:
They are not relavant for other data types as far as I know. I separated them because they get updated independantly from the SPIR-V file layout. I can merge them back if needed :)

Reply:
I have updated the PR & merged the enums to the spirv.hexpat :)